linode_api4/groups/placement_groups.py

```diff
@@ -0,0 +1,68 @@
+from typing import Union
+
+from linode_api4.errors import UnexpectedResponseError
+from linode_api4.groups import Group
+from linode_api4.objects.placement_groups import PlacementGroup
+from linode_api4.objects.region import Region
+
+
+class PlacementAPIGroup(Group):
```

Comment:
I'm not quite sure what to name this class given the convention here would be `{route_name}Group`, but that conflicts with `PlacementGroup`